README.rst

```diff
@@ -235,6 +236,44 @@ Delete
     print(response)
     # <class pybutton.Response >
 
+Customers
+~~~~~~
+
+Create
+''''''
+
+.. code:: python
+
+    import hashlib
+    from pybutton import Client
+
+    client = Client('sk-XXX')
+
+    hashed_email = hashlib.sha256('user@example.com'.lower().strip()).hexdigest()
+
+    response = client.customers.create({
+        'id': 'customer-1234',
+        'email_sha256': hashed_email,
+        'advertising_id': '6E82078A-8146-4BA4-AC5B-76104861E61A',
+    })
+
+    print(response)
+    # <class pybutton.Response id: 'customer-1234', ...>
```

Comment:
Yeah, I was just getting confused thinking you were signaling the internal identifier, not the external.  Carry on.